packages/client/hmi-client/src/assets/css/theme/_fonts.scss

```diff
@@ -21,4 +21,48 @@ h5 {
 
 h6 {
     font-size: $fontBodySmall;
-} 
+}
```

Comment:
Should this be in it's own file, or is it alright in this file?

Reply:
This is the right place! Thanks for checking in.